sources/packages/backend/libs/test-utils/src/factories/announcement.ts

```diff
@@ -0,0 +1,22 @@
+import { Announcement } from "@sims/sims-db";
+import * as faker from "faker";
+
+export function createFakeAnnouncement(
+  startDate: Date,
+  endDate: Date,
+  messageTitle?: string,
+  message?: string,
+  target?: string[],
```

Comment:
To pass the values of the factory object please use initial values like this. 

```ts

export function createFakeAnnouncement(options?: {
  initialValues?: Partial<Announcement>;
}): Announcement {
  const announcement = new Announcement();
  announcement.message =
    options?.initialValues?.message ?? faker.random.words(3);
  announcement.messageTitle =
    options?.initialValues?.messageTitle ?? faker.random.words(10);
  announcement.target = options?.initialValues?.target ?? [
    "student-dashboard",
    "institution-dashboard",
  ];
  announcement.startDate = options?.initialValues?.startDate;
  announcement.endDate = options?.initialValues?.endDate;

  return announcement;
}
```

Reply:
updated